src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/Microsoft.AspNetCore.Razor.LanguageServer.Common.csproj

```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonPackageVersion)" />
+    <PackageReference Include="OmniSharp.Extensions.LanguageProtocol" Version="$(OmniSharpExtensionsLanguageProtocolPackageVersion)" />
```

Comment:
I assume this is here because of the JsonConverter pieces. Could we move those from the Common project to the LanguageServer project? I ask because this project is used by O#'s plugin and our language server. In the plugin world we don't need the converters you added.